pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -67,8 +48,8 @@ enum pokemonStatIndex {
 class PokemonDTO {
   public name: string;
   public dexId: number;
-  public type1: string; //TODO: Update this to a TypeDTO object when created
-  public type2: string | null;
+  public type1: ElementType;
+  public type2: ElementType;
```

Comment:
Now that there is an `ElementType.NULL`, there is no need to allow for a `null` in a Pokemon's second type.